### PR TITLE
8 sale team multicompany

### PR DESCRIPTION
**Pull request by chafique-delli**
Add companies in sales team and filter the sales team from the company

### CODE REVIEW

**Comment by bealdav** [comment]
Seems Ok, but please add demo data + small test unit for create sale and check than sale is not accessible for none authorised company.
Inspiration example: https://github.com/OCA/product-attribute/blob/8.0/pricelist_per_product/tests/test_create.py

**Comment by sebastienbeau** [comment]
Hi Chafique, as discuss with David it will be better to add demo data and use this demo data in the test to check that everything is ok. The main advantage is that if somebody want to trie the module it's easier to test it as there is demo data

File: sale_team_multicompany/tests/test_sales_team_multicompany.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# © 2016 Chafique DELLI @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+from openerp.osv.orm import except_orm as AccessError
```
Comment (bealdav) [inline sale_team_multicompany/tests/test_sales_team_multicompany.py]: Y a rien dans openerp.exceptions ?

**Comment by bealdav** [comment]
:+1:  testé et approuvé
